revoke/revoke.go

```diff
@@ -288,12 +285,13 @@ func sendOCSPRequest(server string, req []byte, issuer *x509.Certificate) (*ocsp
 	}
 	resp.Body.Close()
 
-	if bytes.Equal(body, ocspUnauthorised) {
-		return nil, errors.New("OSCP unauthorized")
-	}
-
-	if bytes.Equal(body, ocspMalformed) {
-		return nil, errors.New("OSCP malformed")
+	switch {
+	case bytes.Equal(body, ocsp.UnauthorizedErrorResponse),
+		bytes.Equal(body, ocsp.MalformedRequestErrorResponse),
+		bytes.Equal(body, ocsp.InternalErrorErrorResponse),
+		bytes.Equal(body, ocsp.TryLaterErrorResponse),
+		bytes.Equal(body, ocsp.SigRequredErrorResponse):
+		return nil, err
```

Comment:
This is incorrect. err is nil. Sorry we can't merge this if this change doesn't come with tests.